test/models/erc1155-standard.spec.ts

```diff
@@ -0,0 +1,68 @@
+import {defaultWeb3Connection, getPrivateKeyFromFile,} from '../utils/';
+import {ERC1155Standard, Web3Connection} from '../../src';
+import {expect} from 'chai';
+
+describe(`ERC1155 Standard`, () => {
+  let contract: ERC1155Standard;
+  let web3Connection: Web3Connection;
+  let contractAddress: string;
+  const initial = {
+    uri: 'https://my.token.uri'
+  }
+  before(async () => {
+    try {
+      web3Connection = await defaultWeb3Connection(true, true);
+    } catch (error) {
```

Comment:
you don't want to catch the error on the before: if the web3connection failed to establish for any reason the test should fail as nothing can exist without a stable web3connection